tests/test_visualization.py

```diff
@@ -0,0 +1,19 @@
+import pytest
+import numpy as np
+import numpy.testing as npt
+from tinnsleep.data import RawToEpochs_sliding,CreateRaw
+from tinnsleep.visualization import create_visual
+from tinnsleep.utils import compute_nb_epochs
+
+
+def test_visualization():
+    data = np.random.randn(2, 400)
+    ch_names = ['Fz', 'Pz']
+    duration = 0.1 #in seconds
+    interval =0.1  #in seconds
+    raw = CreateRaw(data, ch_names)
+    leny=compute_nb_epochs(400, 0.1*200, 0.1*200)  #important : convert the 
+    #duration and interval in number of samples
+    b_detect=[False for i in range(leny)]
+    b_detect[2]=True
+    create_visual(raw, b_detect, duration, interval)
```

Comment:
- avoid to display fig during pytest. There are several way to do that, but you need to insure that create_visual doesn't not print by default. 
- It is also better to add a parameter "figure=None" as input of any function using plt. It allows to print your figure in any figure or subplot given by the user. (if None, you create a new one using `figure=plt.figure`). but here it doesn't work because you generate two figures
- avoid making TWO figures with one function. Use subplots if you need to do it in one function. Obviously here it is a problem because we use raw.plt() which is pretty bad. I'll add an issue to do our own `plotTimeSeries()`function.
- using pytest fixtures with matplotlib should be better https://www.tutorialspoint.com/pytest/pytest_fixtures.htm but I don't know yet how to do that so we skip that for now

Reply:
PEP8: one blank line at the end of file.

Reply:
- I didn't know about "figure = None", very useful! I'll try to remember for future use!
- We can definitely think about spliting the function into two, with one figure in each... It would be smarter, I'll try to do it!
- I have often been disapointed by the visualization tool of mne, but it has the advantage of making possible the annotation/event process, which is quite visual. Is there an equivalent on plotTimeSeries()?